circuits/circom/messageHasher.circom

```diff
@@ -13,14 +13,37 @@ template MessageHasher() {
     // we output an hash 
     signal output hash;
 
-    component hasher = Hasher13();
+    // Hasher5(
+    //     in[0]
+    //     Hasher5_1(in[1], in[2], in[3], in[4], in[5]),
+    //     Hasher5_2(in[6], in[7], in[8], in[9], in[10])
+    //     in[11],
+    //     in[12]
+    // )
```

Comment:
will we delete this?

Reply:
This is useful to understand how Hasher13 works under the hood (ie., a combination of Hasher5).